templates/template-post.html

```diff
@@ -44,7 +44,7 @@
       <!-- remove advertisement stylization-->
       {% if item.post_items_type != "advertisement" %}
   
-      <div class="lb-post-date relativeDate" data-js-timestamp="{{ postDate }}">{{ postDate }}</div>
+      <div class="lb-post-date relativeDate mod--displaynone" data-js-timestamp="{{ postDate }}">{{ postDate | date('') }}</div>
```

Comment:
Why do you want to hide the date up-front? And why can't we use the date format from the theme settings here?

Reply:
This is for server side render and for that it is used server timezone,
and not browser timezone and when the browser will render it can be different so just for not showing different time  we will not show. 
But just for blogs that run just across a single timezone we will add a theme setting for this.